src/main/java/ma/boot/springboot/service/WordService.java

```diff
@@ -0,0 +1,17 @@
+package ma.boot.springboot.service;
+
+import java.util.List;
+import ma.boot.springboot.model.Word;
+import ma.boot.springboot.model.dto.WordResponseDto;
+
+public interface WordService {
```

Comment:
I think we can make it extend GenericService?

Reply:
The issue is with add method - in Generic we have as an input - ReviewDto, as an output - Model. 
In case of Word model -  if we have Review as an input - we will have a List of Words. Probably we need this but the output will be like the first Word. I'm doubted I need method get and add at all in Word Service. I would rather delete both methods in Service...